datasets/batch_view/first_shutdown_summary/intro.md

```diff
@@ -0,0 +1,30 @@
+The `first_shutdown_summary` table is a summary of the [`first-shutdown`
+ping](https://firefox-source-docs.mozilla.org/toolkit/components/telemetry/telemetry/data/first-shutdown-ping.html). 
+
+#### Contents
+
+The first shutdown ping contains a client's first session usage data. This
+table contains rows similar to the
+[`telemetry_new_profile_parquet`](/datasets/batch_view/new_profile/reference.md),
+but in the shape of
+[`main_summary`](/datasets/batch_view/main_summary/reference.md). 
+
+#### Background and Caveats
+
+Data collection latency was vastly improved with the introduction of the
+shutdown ping-sender mechanism in Firefox 55. To maintain consistent historical
+behavior, the first main ping is not sent until the second start up. Instead, a
+separate first-shutdown ping was created to maintain backwards compatibility.
+
+In many cases, the first-shutdown ping is a duplicate of the main ping. To
+avoid duplicate documents, take the `outer join` of the two datasets on
+their `document_id`.
+
+#### Accessing the Data
```

Comment:
Is this dataset backfilled? If not how far back is the data available?

Reply:
It's backfilled to 20170922, but it's probably most useful from Firefox 57 beta onward.